fabric-rendering-v1/src/client/java/net/fabricmc/fabric/mixin/client/rendering/WorldRendererMixin.java

```diff
@@ -80,6 +80,18 @@ private void beforeRender(ObjectAllocator objectAllocator, RenderTickCounter tic
 	@Inject(method = "setupTerrain", at = @At("RETURN"))
 	private void afterTerrainSetup(Camera camera, Frustum frustum, boolean hasForcedFrustum, boolean spectator, CallbackInfo ci) {
 		context.setFrustum(frustum);
+	}
+
+	@Inject(
+			method = "method_62214",
+			at = @At(
+				value = "INVOKE",
```

Comment:
Would this not be a good place to use `INVOKE_STRING`? That way this can specifically target the `terrain` profiler push call, even if it gets moved around